shard-operator-parent/shard-operator-core/src/main/java/com/redhat/service/smartevents/shard/operator/core/networking/KubernetesNetworkingService.java

```diff
@@ -75,11 +76,13 @@ public boolean delete(String name, String namespace) {
         }
     }
 
-    private Ingress buildIngress(BridgeIngress bridgeIngress, Service service, Integer port, String path) {
+    private Ingress buildIngress(HasMetadata bridgeIngress, Service service, Integer port, String path) {
```

Comment:
The parameter name suggests you expect a `bridgeIngress`, but you accept an `HasMetadata`. You might want to rename `bridgeIngress` to `resource`. If the `buildIngress` method is specific for a `bridgeIngress`, I suggest to keep the `BridgeIngress` and not a generic `HasMetadata` resource

Reply:
Will rename to `resource` 👍